packages/cuba-react-ui/jest.config.js

```diff
@@ -21,5 +21,9 @@ module.exports = {
   },
   "collectCoverageFrom": [
     "src/**/*.{ts,tsx}",
-  ]
+  ],
+  "moduleNameMapper": {
+    "^react$": "<rootDir>/../cuba-react-core/node_modules/react",
+    "^mobx-react$": "<rootDir>/../cuba-react-core/node_modules/mobx-react"
+  }
```

Comment:
The reason behind this: due to the fact that `cuba-react-core` is symlinked, it will use it's own instance of React, while the tests will use the one from `cuba-react-ui/node_modules`. In case of `FieldPermissionContainer.test.tsx` that would result in an [Invalid Hook Call](https://reactjs.org/warnings/invalid-hook-call-warning.html#duplicate-react) when the test transitively imports `MainStore`, which declares a custom hook.

Reply:
May be add this as comment inside jest.config.js file?